components/builder-depot/src/data_store.rs

```diff
@@ -133,20 +133,18 @@ impl PackagesIndex {
         Ok(val)
     }
 
-    pub fn list(&self,
-                id: &str,
-                offset: isize,
-                count: isize)
-                -> Result<Vec<depotsrv::PackageIdent>> {
+    pub fn list(&self, id: &str, start: isize, stop: isize) -> Result<Vec<depotsrv::PackageIdent>> {
         let conn = self.pool().get().unwrap();
-        match conn.zrange::<String, Vec<String>>(Self::key(&id.to_string()), offset, count) {
+
+        // Note: start and stop are INCLUSIVE ranges
+        match conn.zrange::<String, Vec<String>>(Self::key(&id.to_string()), start, stop) {
             Ok(ids) => {
                 // JW TODO: This in-memory sorting logic can be removed once the Redis sorted set
                 // is pre-sorted on write. For now, we'll do it on read each time.
                 let mut ids: Vec<package::PackageIdent> =
                     ids.iter().map(|id| package::PackageIdent::from_str(id).unwrap()).collect();
                 ids.sort();
-                ids.reverse();
+                // ids.reverse();
```

Comment:
Hi - I'm not quite sure why it was reversed in the first place, however the story acceptance criteria calls for the ordering to not be reversed as it is today.